neat_core.c

```diff
@@ -434,7 +451,12 @@ he_connected_cb(uv_poll_t *handle, int status, int events)
         flow->sockType = he_ctx->candidate->ai_socktype;
         flow->sockProtocol = he_ctx->candidate->ai_protocol;
         flow->everConnected = 1;
+#if defined(USRSCTP_SUPPORT)
+        flow->sock = he_ctx->sock;
+#else
         flow->fd = he_ctx->fd;
+#endif
+
```

Comment:
I'm not sure here, but don't you assume that if USRSCTP_SUPPORT is defined all sockets are handled via the library? This is not true, since you would still use kernel sockets for TCP, UDP and UDPLite. I guess you have to check that.